driver_go1.10.go

```diff
@@ -0,0 +1,20 @@
+// Go MySQL Driver - A MySQL-Driver for Go's database/sql package
+//
+// Copyright 2018 The Go-MySQL-Driver Authors. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// +build go1.10
```

Comment:
I think the build tags are superfluous, see #822

Reply:
Sorry, I was wrong in #822. The build flag is necessary.